File: deaf_grandma.js
function deafGrandma(greeting) {
  var text;

let response = prompt("say something to your grandma");

  switch(response) {
  case "":
    text = "WHAT?!";
    break;
  case response.toLowerCase();
    text = "SPEAK UP, KID!";
    break;
  case response.toUpperCase():
    text = "NO, NOT SINCE 1946!";
    break;
    case "Goodbye":
    text = "LEAVING SO SOON?";
    break;
  default:
    text = "LATER, SKATER!";
    break;
}
}

deafGrandma()

File: deaf_grandma.py
def deaf_grandma():

  print("Say something to grandma.\n")  
  farewell = "GOODBYE!"
  counter = 0

  while counter < 2:
    prompt = input()
    if prompt == "":
      print("WHAT?!\n")
    elif prompt == prompt.upper() and prompt != farewell:
      print("NO, NOT SINCE 1946!\n")
    elif prompt == prompt.lower():
      print("SPEAK UP, KID!\n")
    elif prompt == farewell and counter == 0:
      print("LEAVING SO SOON?\n")
      counter = 1
    elif prompt == farewell and counter == 1:
      print("LATER, SKATER!")
      counter = 2
    else:
      print("...HUH?!  WHAT?!") 
